index.html

```diff
@@ -2975,45 +2976,38 @@ <h3>Zero-Knowledge Proofs</h3>
         </ul>
 
         <p>
-This specification describes a data model that supports zero-knowledge proof
-mechanisms. The examples below highlight how the data model can be used to
-issue, present, and verify zero-knowledge <a>verifiable credentials</a>.
+This specification describes a data model that supports selective disclosure
+with the use of a zero-knowledge proof mechanisms. The examples below highlight
+how the data model can be used to issue, present, and verify zero-knowledge
+<a>verifiable credentials</a>.
         </p>
 
         <p>
 To use zero-knowledge <a>verifiable credentials</a> the <a>issuer</a> must
-issue a <a>verifiable credential</a> in a manner that enables the <a>holder</a>
-to present the information to a <a>verifier</a> in a privacy-enhancing manner.
+issue a <a>verifiable credential</a> in a manner that enables
+the <a>holder</a> to derive a proof from the original issued <a>verifiable
+credential</a> so that the <a>holder</a> can present the information to a
+<a>verifier</a> in a privacy-enhancing manner.
 This implies that the <a>holder</a> can prove the validity of the
 <a>issuer's</a> signature without revealing the values that were signed, or when
 only revealing certain selected values. The standard practice is to do so by
 proving knowledge of the signature, without revealing the signature itself.
 There are two requirements for <a>verifiable credentials</a> when they are to be
 used in zero-knowledge proof systems. The <a>verifiable credential</a> MUST
-contain a:
-        </p>
-
-        <ul>
-          <li>
-<a>Credential</a> definition, using the <code>credentialSchema</code>
-<a>property</a>, that can be used by all parties to perform various
-cryptographic operations in zero-knowledge.
```

Comment:
instead of getting rid of this, I would prefer it changed from MUST to MAY

Reply:
I'm good with that, will update it.

Reply:
updated in latest commit. Resolving this comment.